code/script.js

```diff
@@ -15,37 +20,190 @@ const showMessage = (message, sender) => {
+        drinkType = "Flat White"; 
+        showMessage("Flat White is my jam!", 'user');
+        setTimeout(milkOptions, 1500);
+      });
+      document.getElementById("cappuccino").addEventListener("click", () => {
+        drinkType = "Cappuccino"; 
+        showMessage("I've never said 'no' to a Cappuccino!", 'user');
+        setTimeout(milkOptions, 1500);
+      });
+}
+
+  // When the drink is chosen, the bot asks the user which milk they want for their drink.
+const milkOptions = () => {
+  showMessage(
+    `A ${drinkType}? Lovely! What kind of milk would you like us to prepare it with?`,
+    'bot');
+    // Buttons for the different milk choices
+      inputWrapper.innerHTML = `
+      <button id="oatMilk">Oat milk</button>
+      <button id="almondMilk">Almond milk</button>
+      <button id="soyMilk">Soy milk</button>`
+
+        document.getElementById("oatMilk").addEventListener("click", () => {
+          milkType = "Oat milk"; 
+          showMessage("Oat milk, please!", 'user');
+          setTimeout(confirmMilk, 1500);
+        });
+        document.getElementById("almondMilk").addEventListener("click", () => {
+          milkType = "Almond milk"; 
+          showMessage("Almond milk, please!", 'user');
+          setTimeout(confirmMilk, 1500);
+        });
+        document.getElementById("soyMilk").addEventListener("click", () => {
+          milkType = "Soy milk"; 
+          showMessage("Soy milk, please!", 'user');
+          setTimeout(confirmMilk, 1500);
+        });
+}
+
+// Bot confirms the choice of milk to the user
+const confirmMilk = () => {
+  showMessage(`You chose to have your drink with ${milkType}. Great choice!`, 'bot');
+  setTimeout(sizeChoice, 2000);
+}
+
+
+// Bot asks the user which size of the drink they want
+const sizeChoice = () => {
+  showMessage(
+    `Which drink size would you prefer today: big or small?`,
+    'bot')
+    // Buttons for the drink sizes
+      inputWrapper.innerHTML = `
+      <button id="bigDrink">Big</button>
+      <button id="smallDrink">Small</button>`
+
+        document.getElementById("bigDrink").addEventListener("click", () => {
+          drinkSize = "Big"; 
+          showMessage("Big! I need my caffeine.", 'user');
+          setTimeout(confirmSize, 1500);
+        });
+        document.getElementById("smallDrink").addEventListener("click", () => {
+          drinkSize = "Small"; 
+          showMessage("Small. It's almost my bedtime...", 'user');
+          setTimeout(confirmSize, 1500);
+        });
+}
+
+// Bot confirms the user's choice of drink size
+const confirmSize = () => {
+  if (drinkSize === "Big") {
+    showMessage("A big ol' cuppa coming right up!", 'bot');
+    setTimeout(feedback, 2000);
+  } else if (drinkSize === "Small") {
+    showMessage("I understand! Let's make you just a small one then.", 'bot');
+    setTimeout(feedback, 2000);
+  } else {
+    showMessage("Hmm, I believe that's not an option.", 'bot');
```

Comment:
`  } else {
    showMessage("Hmm, I believe that's not an option.", 'bot');`
Is this necessary? If I'm not mistaken there was only two buttons to choose from, and there wouldn't be any way of invoking the "else"..? :)